README.md

```diff
@@ -18,29 +18,29 @@
 
 ---
 
-- [Content](#content)
-  - [Content Files](#content-files)
-  - [External Content Files](#external-content-files)
-  - [Markdown File Requirements](#markdown-file-requirements)
-  - [Adding Docs](#adding-docs)
-  - [Editing Docs](#editing-docs)
-  - [Repositories](#repositories)
-    - [Add Links to a Repository](#add-links-to-a-repository)
-    - [Release Versions](#release-versions)
-  - [API References](#api-references)
-    - [Swagger specs](#swagger-specs)
-    - [TypeDoc specs](#typedoc-specs)
-- [Development](#development)
-  - [Using npm](#using-npm)
-  - [Using Docker](#using-docker)
-- [Linting & Formatting](#linting--formatting)
-  - [Editor Setup: VS Code](#editor-setup-vs-code)
-- [GitHub Data Fetching](#github-data-fetching)
-  - [GitHub GraphQL API](#github-graphql-api)
-  - [GitHub REST API](#github-rest-api)
-- [Deployment](#deployment)
-- [Authors](#authors)
-- [License](#license)
+- [Content](#Content)
+  - [Content Files](#Content-Files)
```

Comment:
Did GitHub change how they name the internal anchor links? That is going to break so many Table of Contents...